src/browserlib/extract-references.js

```diff
@@ -51,12 +51,6 @@ function getExtractionRules(generator) {
         informative: "#informative-references > dl"
       }
     },
```

Comment:
Nit: Could you drop the trailing comma here?

Reply:
Done (force pushed)